syntax = "proto3";

package auth;

service AuthService {
    rpc LoginUser(UserCredentials) returns (TokenResponse);
}

message UserCredentials {
    string email = 1;
    string password = 2;
}


message TokenResponse {
    string token = 1;
}


// message UserData {  // сейчас не используется
//     map<string, string> data = 1; // пример словаря в данных
// }

// message AdminRegisterCredentials(
//     string email_admin = 1;
//     string password_admin = 2;
//     UserData user_data = 3;
    // repeated string user_data = 3; это типо список строк, если бы я хотел чтобы именно списком прилетали какие либо даннные.
//     )


